build.sbt

```diff
@@ -113,8 +113,8 @@ lazy val core = project
     librarySettings,
     name := s"$baseName-core",
     libraryDependencies ++= Seq(
-      "com.typesafe.akka" %% "akka-actor"        % akkaVersion % Provided,
-      "com.typesafe.akka" %% "akka-stream"       % akkaVersion % Provided,
+      "com.typesafe.akka" %% "akka-actor"        % akkaVersion,
```

Comment:
`Provided` is removed here because otherwise we can't enforce the correct version of akka/akka-streams which we need since its only with akka `2.6.19` and above where we have `unsafeDataVia` which is required for this feature (see https://github.com/akka/akka/pull/31123)